src/js/gameBuilder.js

```diff
@@ -4,38 +4,34 @@ import {
     onTouchStart,
     onTouchEnd
  } from "./actions.js";
+import { removeGameTable } from "./gameState";
 
-export function buildGameTable(level) {
-    let rows = level.rows;
-    let cols = level.cols;
-    let maxMines = level.maxMines;
-    let tableCells = {};
-    let table = {
+const fragment = document.createDocumentFragment();
+const gameTableElement = document.getElementById("gameTable");
+
+gameTableElement.addEventListener("click", clickOnCell);
+gameTableElement.addEventListener("contextmenu", rightClickOnCell, false);
+gameTableElement.addEventListener('touchstart', onTouchStart,false);
+gameTableElement.addEventListener('touchend', onTouchEnd,false);
+
+export function buildGameTable([rows, cols, maxMines]) {
+    let tableCells = Object.create(null);
+    let table = Object.create(null);
+    let cellClass = 'cell-' + rows;
+
+    Object.assign(table, {
         counterMines:0, 
         maxMines: maxMines, 
         counterFlags:0, 
         amountCells: rows * cols, 
         timeCounterIsStart: false, 
-        firstClick: false
-    };
-    let gameTableElement = document.getElementById("gameTable");
-    document.getElementById("amount-mines").innerHTML = maxMines;
+        firstClick: false,
+        won: undefined
+    });
 
-    let cellClass;
-    switch (rows) {
-        case 8:
-            cellClass = "cell-8"
-            break;
-        case 10:
-            cellClass = "cell-10"
-            break;        
-        case 12:
-            cellClass = "cell-12"
-            break;
-        default:
-            break;
-    }
-    
+    removeGameTable();
```

Comment:
Putting this here is in my opinion quite logic and makes `app.js` easier to handle.